core/.gitignore

```diff
@@ -0,0 +1,22 @@
+extra.mk
+extra.dpdk.mk
+
+# Executables
+bessd
+
+# Unittest/benchmark executables
+test_all
```

Comment:
If I'm not mistaken, `test_all` doesn't exist anymore, we only have `all_test`, right?

Reply:
Thanks! All updated.